### PR TITLE
fix #10: call updateAssetStateInternal function

### LINKED ISSUE #10: updateAssetStateInternal update issue 2
- [x] test case 3
- [x] call updateAssetStateInternal function 
- [x] test case 1 again
- [x] test case 2 again
- [x] test case 3 again

### PR DESCRIPTION
**Pull request by rkdud007**

### CODE REVIEW

**Comment by rkdud007** [comment]
try3 : https://goerli.etherscan.io/address/0x254a37E63EB18A37C6B1fbAb9F6E57D0eD23aDb5

**Comment by rkdud007** [comment]
after modify
test 2 : https://goerli.etherscan.io/address/0xF2B3F85b8E83C4EAfcEbBD0c4c0175C38d2c2da9

**Comment by rkdud007** [comment]
after modify 
test 3 : https://goerli.etherscan.io/address/0xba9c5Ca76bCd1b69c322920F3775616D07dC8216

File: contracts/contracts/StakingRewards.sol [hunk 1]
```diff
@@ -200,7 +200,7 @@ contract StakingRewards is Initializable {
   function configEmissionPerSecond(
     uint128 _emissionPerSecond
   ) external onlyOperator {
-
+    updateAssetStateInternal ();
```
[the remark this answers is not in the document]
Comment (rkdud007) [reply contracts/contracts/StakingRewards.sol]: fix

**Review by ahramyun7 — APPROVED** [review]
LGTM, approved

**Pull request merged**